tools/gstf_preparation/gstf_preparation.py

```diff
@@ -139,10 +149,15 @@ def add_transcript_to_dict(cols, species, transcript_dict):
     transcript = feature_to_dict(cols)
     if 'biotype' in transcript and transcript['biotype'] != 'protein_coding':
         return
```

Comment:
At the moment this filtering of gene and transcripts is applied only to GFF3 files, not gene features in JSON format. Do we want to extend it to JSON files too?

Reply:
Yes.. I will do that...

Reply:
Done.